components/Chips/examples/ChipsActionExampleViewController.m

```diff
@@ -0,0 +1,116 @@
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "ChipsExamplesSupplemental.h"
+
+#import "MaterialChips.h"
+#import "MDCChipViewColorThemer.h"
+
+@implementation ChipsActionExampleViewController {
+  UICollectionView *_collectionView;
+  MDCChipView *_sizingChip;
+  MDCSemanticColorScheme *_colorScheme;
+}
+
+- (void)loadView {
+  [super loadView];
+
+  _colorScheme = [[MDCSemanticColorScheme alloc] init];
+
+  // Our preferred CollectionView Layout For chips
+  MDCChipCollectionViewFlowLayout *layout = [[MDCChipCollectionViewFlowLayout alloc] init];
+  layout.minimumInteritemSpacing = 10;
+
+
+  // Action chips should allow single selection, collection view default is based on signle
+  // selection. Note that MDCChipCollectionViewCell manages the state of the chip accordingly.
+  _collectionView = [[UICollectionView alloc] initWithFrame:CGRectZero collectionViewLayout:layout];
+
+  // Since there is no scrolling turning off the delaysContentTouches makes the cells respond faster
+  _collectionView.delaysContentTouches = NO;
+
+  // Collection view setup
+  _collectionView.dataSource = self;
+  _collectionView.delegate = self;
+  _collectionView.backgroundColor = [UIColor whiteColor];
+  _collectionView.contentInset = UIEdgeInsetsMake(4, 8, 4, 8);
+  [_collectionView registerClass:[MDCChipCollectionViewCell class]
+      forCellWithReuseIdentifier:@"Cell"];
+
+  // This is used to calculate the size of each chip based on the chip setup
+  _sizingChip = [[MDCChipView alloc] init];
+
+  [self.view addSubview:_collectionView];
+}
+
+- (void)viewWillLayoutSubviews {
+  [super viewWillLayoutSubviews];
+
+  _collectionView.frame = self.view.bounds;
+}
+
+- (NSInteger)collectionView:(UICollectionView *)collectionView
+     numberOfItemsInSection:(NSInteger)section {
+  return self.titles.count;
+}
+
+- (__kindof UICollectionViewCell *)collectionView:(UICollectionView *)collectionView
+                           cellForItemAtIndexPath:(NSIndexPath *)indexPath {
+  MDCChipCollectionViewCell *cell =
+  [collectionView dequeueReusableCellWithReuseIdentifier:@"Cell" forIndexPath:indexPath];
+  MDCChipView *chipView = cell.chipView;
+
+  // Customize Chip
+  chipView.titleLabel.text = self.titles[indexPath.row];
+
+  // Apply Theming
+  [MDCChipViewColorThemer applySemanticColorScheme:_colorScheme toChipView:cell.chipView];
+
+  return cell;
+}
+
+- (void)collectionView:(UICollectionView *)collectionView didSelectItemAtIndexPath:(NSIndexPath *)indexPath {
+  // For action chips, we never want the chip to stay in selected state.
+  // Other possible apporaches would be relying on theming or Customizing collectionViewCell
+  // selected state.
+  [collectionView deselectItemAtIndexPath:indexPath animated:NO];
```

Comment:
I'd expect this to be `animated:YES` - was there a technical reason why this had to be NO?

Reply:
oh for actions we don't want user to see the selected state. so it shouldn't be animated.

Reply:
I think it makes more sense for this to live inside the themer though, technically theming action chip in a way that there is no selected state. I need to figure that out with designers and then update this.

Reply:
Hmm let's be careful that we're not removing user touch feedback. At a very minimum we're still showing the ink ripple when a chip is selected, correct?

Reply:
Ink will be spread in highlighted, and we still animate selected.